content/api/events.apib

```diff
@@ -0,0 +1,269 @@
+        + Default: `1000`
+    + delimiter (optional, string) ... Specifies the delimiter for query parameter lists.
+        + Default: `,`
+    + events (string, optional) - Comma delimited list of event types to search.
+        + Default: `all event types`
+    + recipients (string, optional, `recipient@example.com`) ... Comma delimited list of recipients to search.
+    + recipient_domains (string, optional) ... Comma delimited list of recipient domains to search. Supports wildcarding.
+    + from_addresses (string, optional) - Comma delimited list of friendly from addresses to search.
+    + sending_domains (string, optional) - Comma delimited list of sending domains to search. Supports wildcarding.
+    + subject (string, optional) - Subject line from the email header to search. Supports wildcarding.
+    + bounce_classes (number, optional) - Comma delimited list of bounce classification codes to search. See [Bounce Classification Codes](https://www.sparkpost.com/docs/deliverability/bounce-classification-codes/).
+    + reason (optional, string) ... Bounce/failure/rejection reason to search. Supports wildcarding.
+    + campaigns (string, optional) ... Comma delimited list of campaign IDs to search. Supports wildcarding.
+    + templates (string, optional, `my-template`) ... Comma delimited list of template IDs to search. Supports wildcarding.
+    + sending_ips (string, optional) - Comma delimited list of sending IP addresses to search.
+    + ip_pools (string, optional) - Comma delimited list of IP pool IDs to search. Supports wildarding.
+    + subaccounts (string, optional) - Comma delimited list of subaccount IDs to search.
+    + message_ids (string, optional) - Comma delimited list of message IDs to search.
+    + transmission_ids (string, optional) - Comma delimited list of transmission IDs to search.
+    + ab_tests (string, optional) - Comma delimited list of A/B test IDs to search. Supports wildcarding.
+    + ab_test_version (number, optional) - Version number of A/B test to search. If provided, `ab_tests` parameter becomes required.
+
++ Request
+
+    + Headers
+
+        Accept: application/json
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
++ Response
+    
+        {
+            "results": [
+                {
+                    "type": "click",
+                    "campaign_id": "Example Campaign Name",
+                    "customer_id": "1",
+                    "delv_method": "esmtp",
+                    "event_id": "92356927693813856",
+                    "friendly_from": "sender@example.com",
+                    "geo_ip": {
+                        "country": "US",
+                        "region": "MD",
+                        "city": "Columbia",
+                        "latitude": "39.1749",
+                        "longitude": "-76.8375"
+                    },
+                    "injection_time": "2016-04-18T14:25:07.000Z",
```

Comment:
all timestamp fields should now return in the same format. e.g. `2018-11-29T16:56:27.000+00:00`